modules/cnv-add-nic-to-vm.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/cnv-edit-vms.adoc
+// * cnv/cnv_users_guide/cnv-editing-vm-template
+//
+// Establishing conditionals so content can be re-used for editing VMs
+// and VM templates.
+
+ifeval::["{context}" == "cnv-add-nic-to-vm"]
```

Comment:
This context needs to use the assembly id. In this case, `cnv-edit-vms`

Reply:
Done. Sorry, stupid mistake. I get it, i just forgot to make these changes after the concepts sunk in! :-)

Thanks @aburdenthehand 
Bob